internal/observability/call.go

```diff
@@ -120,14 +120,6 @@ func (c call) endStats(elapsed time.Duration, err error, isApplicationError bool
 			counter.Inc()
 		}
 		return
-	case yarpcerrors.CodeOK:
-		// If we got "CodeOK" it really means that this is not a yarpcError, in
-		// which case this is another level of "unknown" error.
-		c.edge.serverErrLatencies.Observe(elapsed)
-		if counter, err := c.edge.serverFailures.Get("unknown_internal_yarpc"); err == nil {
```

Comment:
we should keep this counter using the "isYarpcError" api

Reply:
Done